.github/workflows/continuous_integration.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: true
       matrix:
         os: ["ubuntu-latest", "windows-latest", "macos-latest"]
-        python-version: ["3.8", "3.10"]
+        python-version: ["3.8", "3.11"]
```

Comment:
We can support Python 3.11 (I tried supporting 3.12 too but ran into problems with the packages).